### PR TITLE
Access control: add a config section for RBAC

### PR DESCRIPTION
**Pull request by IevaVasiljeva**
**What this PR does / why we need it**:
adds a config section for RBAC.

**Which issue(s) this PR fixes**:

Related to https://github.com/grafana/grafana-enterprise/issues/2503

**Special notes for your reviewer**:
The main thing that we should decide on in this PR is how we want to call the new config section. Do you think RBAC makes sense? I thought that "access control" is too generic.

### CODE REVIEW

File: conf/defaults.ini [hunk 1]
```diff
@@ -640,6 +640,10 @@ managed_identity_enabled = false
 # Should be set for user-assigned identity and should be empty for system-assigned identity
 managed_identity_client_id =
 
+#################################### Role-based Access Control ###########
```
Comment (vtorosyan) [inline conf/defaults.ini:643]: Just to double check, when `custom.ini` is provided, does Grafana check first the custom file and fallbacks to defaults.ini for anything not defined in custom one?

**Review by vtorosyan — COMMENTED** [review]
I support naming the section `rbac`. Assumption is that most of the users are going to search for an "rbac" instead of access control, and this is true both for the docs and for the configs.

File: pkg/setting/setting.go [hunk 2]
```diff
@@ -1344,6 +1347,11 @@ func readAuthSettings(iniFile *ini.File, cfg *Cfg) (err error) {
 	return nil
 }
 
+func readAccessControlSettings(iniFile *ini.File, cfg *Cfg) {
+	rbac := iniFile.Section("rbac")
+	cfg.RBACEnabled = rbac.Key("enabled").MustBool(true)
```
Comment (sakjur) [inline pkg/setting/setting.go:1352]: The value here should almost always match the value in `defaults.ini`, fwiw

**Comment by IevaVasiljeva** [comment]
Closing this in favour of https://github.com/grafana/grafana/pull/48813, thanks for the comments all.

**Pull request closed without merging**